<!-- -*- mode: XML -*- -->

<launch>
	
	<node pkg="teresa_wsbs" type="wsbs_controller" name="wsbs_controller" output="screen" respawn="false">
		<param name="xtion_id" value="/depthcamscan_node/scanXtion"/>
		<param name="path_file" value="$(find teresa_wsbs)scenarios/upo45_1.xml"/>
                <param name="heuristic_controller" value="true"/>
	</node>


    

</launch>
